static/fluent/pl/main.ftl

```diff
@@ -0,0 +1,524 @@
+rules-drag-input-hint = Dodaj urządzenie jako wejściowe
+rules-drag-output-hint = Dodaj urządzenie jako wyjściowe
+rules-scroll-left =
+  .alt = Przewiń w lewo
+rules-scroll-right =
+  .alt = Przewiń w prawo
+rules-delete-prompt = Upuść urządzenie tutaj, aby rozłączyć
+rules-delete-dialog = Czy na pewno chcesz trwale usunąć te reguły?
+rules-delete-cancel =
+  .value = Anuluj
+rules-delete-confirm =
+  .value = Usuń Reguły
+rule-invalid = Niepoprawna
+rule-delete-prompt = Czy na pewno chcesz trwale usunąć tę regułę?
+rule-delete-cancel-button =
+  .value = Anuluj
+rule-delete-confirm-button =
+  .value = Usuń Regułę
+rule-select-property = Wybierz Właściwość
+rule-not = Nie
+rule-event = Zdarzenie
+rule-action = Akcja
+rule-configure = Konfiguruj...
+rule-time-title = Godzina
+rule-notification = Powiadomienie
+notification-title = Tytuł
+notification-message = Wiadomość
+notification-level = Poziom
+notification-low = Niski
+notification-normal = Normalny
+notification-high = Wysoki
+rule-name = Nazwa Reguły
+rule-and = i
+rule-or = lub
+rule-time-of-day-is = jest godzina
+rule-event-lower = zdarzenie
+rule-event-occurs = ma miejsce
+
+## Logs
+logs = Wykresy
+logs-create-log-hint = Nie utworzono wykresów. Kliknij +, aby utworzyć wykres.
+logs-device = Urządzenie
+logs-device-select =
+  .aria-label = Rejestrowane Urządzenie
+logs-property = Właściwość
+logs-property-select =
+  .aria-label = Rejestrowana Właściwość
+logs-retention = Retencja
+logs-retention-length = Czas Retencji
+  .aria-label = Rejestrowany Czas Retencji
+logs-retention-unit =
+  .aria-label = Rejestrowana Jednostka Retencji
+logs-hours = Godziny
+logs-days = Dni
+logs-weeks = Tygodnie
+logs-save = Zapisz
+logs-remove-dialog-title = Usuwanie
+logs-remove-dialog-warning = Usunięcie wykresu spowoduje również usunięcie wszystkich jego danych.
+  Czy na pewno chcesz go usunąć?
+logs-remove = Usuń
+logs-unable-to-create = Nie można utworzyć dziennika
+logs-server-remove-error = Błąd serwera: nie można usunąć dziennika
+
+## Add New Things
+add-thing-scanning-icon =
+  .alt = Skanowanie
+add-thing-scanning = Skanowanie w poszukiwaniu nowych urządzeń...
+add-thing-add-adapters-hint = Nie znaleziono żadnych nowych rzeczy. Spróbuj
+add-thing-add-adapters-hint-anchor = dodać rozszerzenia.
+add-thing-add-by-url = Dodaj przez URL...
+add-thing-done = Gotowe
+add-thing-cancel = Anuluj
+
+## Context Menu
+context-menu-choose-icon = Wybierz ikonę...
+context-menu-save = Zapisz
+context-menu-remove = Usuń
+
+## Capabilities
+OnOffSwitch = Przełącznik Włącz/Wyłącz
+MultiLevelSwitch = Przełącznik Wielopoziomowy
+ColorControl = Kontrola Koloru
+ColorSensor = Czujnik Koloru
+EnergyMonitor = Monitor Energii
+BinarySensor = Czujnik Binarny
+MultiLevelSensor = Czujnik Wielopoziomowy
+SmartPlug = Inteligentna Wtyczka
+Light = Światło
+DoorSensor = Czujnik Otwarcia Drzwi
+MotionSensor = Czujnik Ruchu
+LeakSensor = Czujnik Przecieku
+PushButton = Przycisk
+VideoCamera = Kamera Wideo
+Camera = Aparat
```

Comment:
If it's something that records video then it would make more sense to translate as "Kamera". "Aparat" is strongly correlated with taking photos primarily.

Reply:
I was thinking that we have something that records video - Video Camera and something that only take pictures - Camera. Is my thinking wrong?

Reply:
Oh, sorry, looks like you are totally right, Video Camera is just one line above!